### PR TITLE
Newline problem

### PR DESCRIPTION
**Pull request by kvtoraman**
My editor added lots of newlines. I hope that won't be a problem for other people.

### CODE REVIEW

**Review by AnuarTB — CHANGES_REQUESTED** [review]
Don't you think it is better to add Korean language requirements in the Notes column?

**Review by AnuarTB — APPROVED** [review]
Yay, first commit! I think it's a small step for a man, but one huge step for the tech community! :1st_place_medal:

**Comment by kvtoraman** [comment]
I think Korean requirement is a huge change that it needs a table by itself. People may not clearly see "Korean requirement" in notes so they may waste their time, wandering around in the Korean web-page, trying to make it English...

File: README.md [hunk 1]
```diff
@@ -11,33 +11,45 @@ Two-letter locations indicate states in the U.S.; countries other than the U.S.
 
 ## Currently Open Applications
 
+### Companies Require Korean
+
+| Name                                     | Location | Period  | Notes                         | Korean req. |
+| ---------------------------------------- | -------- | ------- | ----------------------------- | ----------- |
+| [Kakao](https://careers.kakao.com/index) | Seoul    | Present | New grad application is over. | Advanced    |
+|                                          |          |         |                               |             |
```
Comment (AnuarTB) [inline README.md]: 19-21 lines - redundant rows.

**Comment by AnuarTB** [comment]
I agree with you on this topic, and I wanted to make language requirements more general to have the possibility to extend to other languages as well in the future. How do you think?

File: README.md [hunk 2]
```diff
@@ -11,7 +11,19 @@ Two-letter locations indicate states in the U.S.; countries other than the U.S.
 
 ## Currently Open Applications
 
+### Companies Require Korean
```
Comment (AnuarTB) [inline README.md]: Companies with specific language requirements

File: README.md [hunk 3]
```diff
@@ -11,7 +11,19 @@ Two-letter locations indicate states in the U.S.; countries other than the U.S.
...
+
+| Name                                     | Location | Period  | Notes                         | Korean req. |
```
Comment (AnuarTB) [inline README.md]: Language req.

File: README.md (hunk 1 above)
Comment (kvtoraman) [reply README.md]: 👍

**Comment by kvtoraman** [comment]
> I agree with you on this topic, and I wanted to make language requirements more general to have the possibility to extend to other languages as well in the future. How do you think?

Alright, let's see how it goes.

**Comment by kvtoraman** [comment]
@AnuarTB  updated.

**Review by AnuarTB — APPROVED** [review]

**Pull request merged**